modules/tensor_mechanics/src/userobjects/CrackFrontDefinition.C

```diff
@@ -81,6 +81,12 @@ addCrackFrontDefinitionParams(InputParameters & params)
                              q_function_type,
                              "The method used to define the integration domain. Options are: " +
                                  q_function_type.getRawNames());
+  params.addParam<UserObjectName>(
+      "crack_front_points_provider",
+      "The UserObject provides the crack front points from XFEM GeometricCutObject");
+  params.addParam<unsigned int>(
+      "number_crack_front_points",
```

Comment:
Same deal here -- maybe this should be changed to something like 'number_points_from_provider' to make it clear that this applies specifically to the case where you're using a provider.